routeros_ip_service  not working

Terraform will perform the following actions:

  # routeros_ip_service.www will be created
  + resource "routeros_ip_service" "www" {
      + certificate = "none"
      + disabled    = true
      + id          = (known after apply)
      + invalid     = (known after apply)
      + name        = (known after apply)
      + numbers     = "www"
      + port        = 80
      + tls_version = "any"
      + vrf         = "main"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

routeros_ip_service.www: Creating...
╷
│ Error: from RouterOS device: unknown parameter
│ 
│   with routeros_ip_service.www,
│   on variables.tf line 22, in resource "routeros_ip_service" "www":
│   22: resource "routeros_ip_service" "www" {
│ 

RouterOS 6.49.10
Expected : 
www service is turned off
<img width="494" alt="image" src="https://github.com/terraform-routeros/terraform-provider-routeros/assets/2346208/a3ce2856-7eec-4af7-812b-75ebc0378fd0">

In Fact - Error

```
locals {
  disable_service = {"ftp" = 21, "telnet" = 23, "www" = 80}
  enable_service  = {"ssh" = 22, "winbox" = 8291}
}

# resource "routeros_ip_service" "disabled" {
#   for_each = local.disable_service
#   numbers  = each.key
#   port     = each.value
#   disabled = true
# }

resource "routeros_ip_service" "www" {
    numbers = "*2"
    port = 80
    disabled = true
}%
╭─anton@Antons-MacBook-Pro-2  ~/GIT/gs-terraform-cloud/tempRouteros ‹node-›  ‹› (main*)
╰─$ ROS_LOG_COLOR=1 TF_LOG=debug terraform apply
2024-04-04T20:19:16.118+0200 [INFO]  Terraform version: 1.5.7
2024-04-04T20:19:16.119+0200 [DEBUG] using github.com/hashicorp/go-tfe v1.26.0
2024-04-04T20:19:16.119+0200 [DEBUG] using github.com/hashicorp/hcl/v2 v2.16.2
2024-04-04T20:19:16.119+0200 [DEBUG] using github.com/hashicorp/terraform-svchost v0.1.0
2024-04-04T20:19:16.119+0200 [DEBUG] using github.com/zclconf/go-cty v1.12.2
2024-04-04T20:19:16.119+0200 [INFO]  Go runtime version: go1.21.1
2024-04-04T20:19:16.119+0200 [INFO]  CLI args: []string{"terraform", "apply"}
2024-04-04T20:19:16.119+0200 [DEBUG] Attempting to open CLI config file: /Users/anton/.terraformrc
2024-04-04T20:19:16.119+0200 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2024-04-04T20:19:16.119+0200 [INFO]  Loading CLI configuration from /Users/anton/.terraform.d/credentials.tfrc.json
2024-04-04T20:19:16.120+0200 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
2024-04-04T20:19:16.120+0200 [DEBUG] ignoring non-existing provider search directory /Users/anton/.terraform.d/plugins
2024-04-04T20:19:16.120+0200 [DEBUG] ignoring non-existing provider search directory /Users/anton/Library/Application Support/io.terraform/plugins
2024-04-04T20:19:16.120+0200 [DEBUG] ignoring non-existing provider search directory /Library/Application Support/io.terraform/plugins
2024-04-04T20:19:16.121+0200 [INFO]  CLI command args: []string{"apply"}
2024-04-04T20:19:16.123+0200 [DEBUG] New state was assigned lineage "4cd3e2e7-e7bd-9736-830f-f9eb1cef7563"
2024-04-04T20:19:16.140+0200 [DEBUG] checking for provisioner in "."
2024-04-04T20:19:16.142+0200 [DEBUG] checking for provisioner in "/opt/homebrew/bin"
2024-04-04T20:19:16.143+0200 [INFO]  backend/local: starting Apply operation
2024-04-04T20:19:16.146+0200 [DEBUG] created provider logger: level=debug
2024-04-04T20:19:16.146+0200 [INFO]  provider: configuring client automatic mTLS
2024-04-04T20:19:16.153+0200 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0 args=[.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0]
2024-04-04T20:19:16.154+0200 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0 pid=61973
2024-04-04T20:19:16.154+0200 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0
2024-04-04T20:19:16.163+0200 [INFO]  provider.terraform-provider-routeros_v1.44.0: configuring server automatic mTLS: timestamp=2024-04-04T20:19:16.163+0200
2024-04-04T20:19:16.171+0200 [DEBUG] provider: using plugin: version=5
2024-04-04T20:19:16.171+0200 [DEBUG] provider.terraform-provider-routeros_v1.44.0: plugin address: address=/var/folders/ps/lcq6km7x6ts2kwg1ntf09frm0000gn/T/plugin2293840467 network=unix timestamp=2024-04-04T20:19:16.171+0200
2024-04-04T20:19:16.188+0200 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2024-04-04T20:19:16.188+0200 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0 pid=61973
2024-04-04T20:19:16.188+0200 [DEBUG] provider: plugin exited
2024-04-04T20:19:16.188+0200 [DEBUG] Building and walking validate graph
2024-04-04T20:19:16.189+0200 [DEBUG] ProviderTransformer: "routeros_ip_service.www" (*terraform.NodeValidatableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2024-04-04T20:19:16.189+0200 [DEBUG] ReferenceTransformer: "routeros_ip_service.www" references: []
2024-04-04T20:19:16.189+0200 [DEBUG] ReferenceTransformer: "local.disable_service (expand)" references: []
2024-04-04T20:19:16.189+0200 [DEBUG] ReferenceTransformer: "local.enable_service (expand)" references: []
2024-04-04T20:19:16.189+0200 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/terraform-routeros/routeros\"]" references: []
2024-04-04T20:19:16.189+0200 [DEBUG] Starting graph walk: walkValidate
2024-04-04T20:19:16.191+0200 [DEBUG] created provider logger: level=debug
2024-04-04T20:19:16.191+0200 [INFO]  provider: configuring client automatic mTLS
2024-04-04T20:19:16.193+0200 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0 args=[.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0]
2024-04-04T20:19:16.194+0200 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0 pid=61974
2024-04-04T20:19:16.194+0200 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0
2024-04-04T20:19:16.199+0200 [INFO]  provider.terraform-provider-routeros_v1.44.0: configuring server automatic mTLS: timestamp=2024-04-04T20:19:16.199+0200
2024-04-04T20:19:16.206+0200 [DEBUG] provider.terraform-provider-routeros_v1.44.0: plugin address: address=/var/folders/ps/lcq6km7x6ts2kwg1ntf09frm0000gn/T/plugin2736855831 network=unix timestamp=2024-04-04T20:19:16.206+0200
2024-04-04T20:19:16.206+0200 [DEBUG] provider: using plugin: version=5
2024-04-04T20:19:16.221+0200 [ERROR] provider.terraform-provider-routeros_v1.44.0: Response contains error diagnostic: diagnostic_attribute=AttributeName("numbers") tf_provider_addr=terraform-routeros/routeros @module=sdk.proto tf_proto_version=5.4 tf_resource_type=routeros_ip_service @caller=github.com/hashicorp/terraform-plugin-go@v0.22.0/tfprotov5/internal/diag/diagnostics.go:58 diagnostic_severity=ERROR diagnostic_summary="Bad value" tf_req_id=9bccd87e-ec74-d56a-3606-a4ae0f2cfe3c tf_rpc=ValidateResourceTypeConfig diagnostic_detail="Unexpected value: *2" timestamp=2024-04-04T20:19:16.221+0200
2024-04-04T20:19:16.221+0200 [ERROR] vertex "routeros_ip_service.www" error: Bad value
╷
│ Error: Bad value
│
│   with routeros_ip_service.www,
│   on main.tf line 32, in resource "routeros_ip_service" "www":
│   32:     numbers = "*2"
│
│ Unexpected value: *2
╵
2024-04-04T20:19:16.221+0200 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2024-04-04T20:19:16.222+0200 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.44.0/darwin_arm64/terraform-provider-routeros_v1.44.0 pid=61974
2024-04-04T20:19:16.222+0200 [DEBUG] provider: plugin exited
```

--- a/routeros/resource_ip_service.go
+++ b/routeros/resource_ip_service.go
@@ -68,8 +68,8 @@ func ResourceIpService() *schema.Resource {
 			Required: true,
 			Description: "The name of the service whose settings will be changed ( api, api-ssl, ftp, ssh, telnet, " +
 				"winbox, www, www-ssl ).",
-			ValidateDiagFunc: ValidationMultiValInSlice([]string{"api", "api-ssl", "ftp", "ssh", "telnet", "winbox",
-				"www", "www-ssl"}, false, false),
+			// ValidateDiagFunc: ValidationMultiValInSlice([]string{"api", "api-ssl", "ftp", "ssh", "telnet", "winbox",
+			// 	"www", "www-ssl"}, false, false),
 		},
 		"port": {
 			Type:         schema.TypeInt,

